### ISSUE TITLE
Problem on the HASS implementation side

### ISSUE THREAD

**Issue by rpitera** [walthowd/ha-alexa-tts#9]
Having an issue on the HASS implementation side... Tests from the command line work fine, no issues with login and tests always result in speech coming out of the appropriate device - tested all of them.

I set up the command line notify objects in my configuration.yaml, restarted HASS and they all show up in services correctly and no errors in the log.

Then I setup an automation to test with and triggered it but nothing happens. No speech and no errors in the HASS log to show why it may have failed.

I also tried it manually from the service dev page (I think I have the JSON correct, but if I don't feel free to point that out):

```
Service: notify.office_dot

Service Data:
{
  "message": "Testing!"
}
```
And again, nothing. No speech, no errors. I see the service call toast message, but nothing happens.

I'm stumped at where to begin troubleshooting and would appreciate any help you can provide.

Further details: My old school HASS root directory is /home/hass and both scripts are in there. Both are properly setup as executable and changing ownership from root to the HASS user doesn't seem to make a difference.

Also tried hard coding the ALEXA_REMOTE variable in alexa_wrapper.sh to reflect my setup's path of /home/hass/alexa_remote_control.sh but that also didn't make a difference.

**Comment by walthowd** [comment]
Trying sending a message without the exclamation point. We don't sanitize user input, and I'm not sure if it would need to be escaped. 

Did you update the PATH variable in alexa_wrapper.sh as specified in the README?

**Comment by rpitera** [comment]
Thanks for replying; I tried without the exclamation point but no change.  I did update the path variable by typing in echo $PATH and copying the results.  It's odd, isn't it?  At least if there was some kind of an error generated I would have a path to follow but nothing gets posted to the log.

I might try this again tomorrow with debug levels on the logs and see if it generates something useful.

Thanks again; I hope I can get this working as TTS for HomeAssistant is the last thing I am keeping the Google Home devices around for!

**Comment by rpitera** [comment]
Well I tried turning on DEBUG log level (what a mess to sort through!) but all I could see was that the automation was being executed.  No errors show.

**Comment by rpitera** [comment]
Also here is my notify section from configuration.yaml
```
- platform: command_line
    name: 'Office Dot'
    command: "/home/hass/alexa_wrapper.sh -d 'Office Dot'" 

  - platform: command_line
    name: 'Kitchen Dot'
    command: "/home/hass/alexa_wrapper.sh -d 'Kitchen Dot'" 

  - platform: command_line
    name: 'Bathroom Dot'
    command: "/home/hass/alexa_wrapper.sh -d 'Bathroom Dot'" 

  - platform: command_line
    name: 'Echo Show'
    command: "/home/hass/alexa_wrapper.sh -d 'Robert's Echo Show" 

  - platform: command_line
    name: 'Bedroom Dot'
    command: "/home/hass/alexa_wrapper.sh -d 'Bedroom Dot'" 

  - platform: command_line
    name: 'Global'
    command: "/home/hass/alexa_wrapper.sh -d 'ALL'" 
```

**Comment by rpitera** [comment]
**SOLVED:**
It turned out to be an error with the names on two devices - one was the Office Dot, which was actually an Echo (I forgot that I moved the dot to the bedroom and replaced it with the Echo once I got the Show) and the other was with the Show which was named "Robert's Echo Show".  I think the issue with this one was the apostrophe, so I just changed the name to "Echo Show" and now they both work.

I feel kinda stupid for not seeing this until today.  Thanks for your help though!
